rumqttc/src/lib.rs

```diff
@@ -1006,10 +1023,4 @@ mod test {
             OptionError::Inflight
         );
     }
-
-    #[test]
-    #[should_panic]
```

Comment:
instead of removing whole test, we should have just removed this `#[should_panic]` attribute.